.github/workflows/databases-nightly.yml

```
@@ -0,0 +1,53 @@
+name: databases-nightly
+
+on:
+  pull_request: # TODO: Remove before merge
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+
+  mariadb:
+    runs-on: ubuntu-22.04
```

```suggestion
    runs-on: ubuntu-22.04
    continue-on-error: true
```

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idcontinue-on-error